src/applications/facility-locator/containers/FacilitesMap.jsx

```diff
@@ -585,6 +585,30 @@ const FacilitiesMap = props => {
     [props.results, map],
   );
 
+  // Handle no results found map transition
+  useEffect(
+    () => {
+      if (
+        props.results &&
+        props.results.length === 0 &&
+        map &&
+        props.currentQuery.searchCoords
+      ) {
+        const { searchCoords } = props.currentQuery;
+        const locationBounds = new mapboxgl.LngLatBounds();
+        const markerElement = buildMarker('currentPos');
+        new mapboxgl.Marker(markerElement)
+          .setLngLat([searchCoords.lng, searchCoords.lat])
+          .addTo(map);
```

Comment:
These 3 lines are repeated 3 times in this file. Consider moving into a helper function?